### ISSUE TITLE
Lost connection to the Cmajor server process!

### ISSUE THREAD

**Issue by andresrumusic** [cmajor-lang/cmajor#88]
<img width="1024" alt="Untitled" src="https://github.com/user-attachments/assets/e8d681f9-4009-4051-87e5-b8daec741016">

So I'm running my first patch. However, I keep getting this message. I can certainly hear what I'm programming, but apparently there's an issue with the connection process to the CMaj server and it cuts it short when running. Can somebody tell me why this happens?

**Comment by cesaref** [comment]
There is one known cause of this, and it is related to having a mac paired to an iphone with the microphone being configured to be shared. This is something we've got an outstanding PR about, and have yet to get to the bottom of. Have a look and see what your default mic and speaker is set to. If this is the case, then I think disabling bluetooth will stop the iphone being used, and everything will work without it.

Let me know if this is the issue.

**Comment by andresrumusic** [comment]
> There is one known cause of this, and it is related to having a mac paired to an iphone with the microphone being configured to be shared. This is something we've got an outstanding PR about, and have yet to get to the bottom of. Have a look and see what your default mic and speaker is set to. If this is the case, then I think disabling bluetooth will stop the iphone being used, and everything will work without it.
> 
> Let me know if this is the issue.

Yup, I think that was the issue here. I restarted my Mac, and it works now! Thank you!
